lib/mayaUsd/fileio/translators/translatorCamera.cpp

```diff
@@ -213,7 +216,12 @@ static bool _TranslateAnimatedUsdAttributeToPlug(
     MTimeArray   timeArray;
     MDoubleArray valueArray;
     if (!_GetTimeAndValueArrayForUsdAttribute(
-            usdAttr, args.GetTimeInterval(), &timeArray, &valueArray, convertToUnit)) {
+            usdAttr,
+            args.GetTimeInterval(),
+            &timeArray,
+            &valueArray,
+            convertToUnit,
+            context->GetTimeSampleMultiplier())) {
```

Comment:
I think we want to allow `context` to be `nullptr` in these translator code paths, so maybe something like this:
```C++
(context != nullptr) ? context->GetTimeSampleMultiplier() : 1.0)) {
```

More on this below.